### ISSUE TITLE
Project status: maintenance mode

### ISSUE THREAD

**Issue by natemcmaster** [natemcmaster/CommandLineUtils#485]
This project is now in maintenance mode. [4.0](https://github.com/natemcmaster/CommandLineUtils/releases/tag/v4.0.0) is the last release I intend to make.

Thank you to the [dozens of contributors over the years](https://github.com/natemcmaster/CommandLineUtils/blob/main/CHANGELOG.md?plain=1#L512-L544).  I believe the work we have done here has produced a highly stable and usable API for building most average command line apps.

**What does 'maintenance mode' mean?**

* Only the most critical of bugs will be fixed (such as security issues).
* No new features will be added.
* I won't be regularly reviewing open issues. Most issues will [timeout and auto-close after 1 year.](https://github.com/natemcmaster/CommandLineUtils/blob/main/.github/workflows/stale.yml).
* I have decided to stop working on this project, but have not put this into "archive mode" on GitHub. Contributors are still welcome to help out by answering others' questions in the issues forum.

**If you need help with this project, please ...**
 * read the documentation (https://natemcmaster.github.io/CommandLineUtils/), 
 * look at the samples (https://github.com/natemcmaster/CommandLineUtils/tree/main/docs/samples),
 * review existing questions (many were answered already) (https://github.com/natemcmaster/CommandLineUtils/issues?q=label%3Aquestion+),
 * or use a programming Q&A forum such as StackOverflow.com
 

Feel free to add comments/questions below. And, thanks again to all the contributors for your work over the years!
-Nate

**Comment by natemcmaster** [comment]
Good suggestion. Updated the readme in #488

**Comment by thohng** [comment]
New library coming `System.CommandLine`, another options for developers.

History: https://github.com/dotnet/command-line-api/blob/f2556cacb35c0b68201d73eaf1c0df8c5e57e43e/docs/History.md

https://www.nuget.org/packages/System.CommandLine

https://learn.microsoft.com/en-us/dotnet/standard/commandline/

**Comment by attilah** [comment]
@thohng yeah it did not age well :-) ....it is still coming and Microsoft does not provide a builtin way to work with command line arguments...

DotNet.CommandLine.CLI is archived all tools should have moved to System.CommandLine by now, EF 8.0 still using this archived library...

Wondering what will happen in this space!

**Comment by bleepzter** [comment]
.NET's System.CommandLine has a really messed up way of integrating with IHostBuilder and IHost. Its like an afterthought.
This library has a much better way of running the command without interfering with DI and the IHostBuilder. Please don't give up on this library because the other one is immature IMHO.

**Comment by FrankRay78** [comment]
Re: 'CommandLine', I contribute to Spectre.Console and I've heard some internal Microsoft teams actually use that, instead of their own 😉
